Source/Core/VideoCommon/HiresTextures.h

```diff
@@ -14,11 +14,15 @@
 
 enum class TextureFormat;
 
+std::set<std::string> GetDirectoriesWithGameid(const std::string& root_directory,
```

Comment:
```suggestion
std::set<std::string> GetDirectoriesWithGameId(const std::string& root_directory,
```

Actually, maybe keeping "texture" in the name (i.e. calling this GetTextureDirectoriesWithGameId) would be better? This is in the global namespace.